docs/mlapi-basics/object-visibility.md

```diff
@@ -4,7 +4,7 @@ title: Object Visibility
 sidebar_label: Object Visibility
 ---
 
-Starting with MLAPI version 6.0.0, clients have no explicit knowledge of all objects or clients that are connected to the server.
+Starting with MLAPI version 0.1.0, clients have no explicit knowledge of all objects or clients that are connected to the server.
```

Comment:
This is actually not correct. Object Visibility was introduced in version 6.0.0 and not 0.1.0.

The thing is we started fresh from the 0.1.0 version when we acquired MLAPI. We will probably need to update/remove comments like this which reference an old and no longer relevant version of MLAPI.